src/core/Form/Select/MultiSelect/MultiSelect/MultiSelect.md

```diff
@@ -148,3 +148,30 @@ const animals = [
     <button onClick={() => setSelectedAnimals([{ labelText: 'Snail', uniqueItemId: 'snail-321' }])}>Snail</button>
 </>
 ```
+
+### Disabled
+
+```
+const foods = [
+  {
+    labelText: 'Pizza',
+    uniqueItemId: 'pizza-123'
+  },
+  {
+    labelText: 'Burger',
+    uniqueItemId: 'burger-321'
+  }
+];
+
+<>
+    <MultiSelect
+      disabled={true}
+      items={foods}
+      labelText="Food"
+      visualPlaceholder="Selection disabled"
+      ariaSelectedAmountText=""
+      ariaOptionsAvailableText=""
+      ariaOptionChipRemovedText=""
```

Comment:
Just a little snag, but maybe we could add some values here and show good example use case with no empty values.
Of course in this case it does not actually matter, but maybe it looks little better and complete example. 🤔

Reply:
Added!